src/app/calculator/utilities/compressed-air-reduction/compressed-air-reduction.component.html

```diff
@@ -0,0 +1,136 @@
+      <div class="d-flex pl-2 align-items-center">
+        <div class="text-center">
+          <span class="fa fa-bolt"></span>
+        </div>
+        <div class="pl-2">
+          <h3>Compressed Air Reduction</h3>
+        </div>
+      </div>
+
+      <div class="d-flex pr-2" *ngIf="inTreasureHunt">
+        <div class="d-flex help-holder">
+          <p class="help-text treasure-hunt-help">Click the save icon to add the treasure to your chest!
+            Add an opportunity sheet at any time or click cancel to discard changes and
+            return to method selection.</p>
+        </div>
+        <div class="pl-2 pr-2 border-right border-left action-item" (click)="addOpportunitySheet()">
+          <span class="fa fa-edit"></span>
+        </div>
+        <div class="pl-2 pr-2 border-right action-item" (click)="save()">
+          <span class="fa fa-save"></span>
+        </div>
+        <div class="pl-2 pr-2 border-right border-left action-item" (click)="cancel()">
+          <span class="fa fa-times"></span>
+        </div>
+      </div>
+    </div>
+
+    <!-- baseline -->
+    <div class="d-flex">
+      <div class="scroll-item p-0 col-8 d-flex bg-white" [ngStyle]="{'height.px': containerHeight}">
+        <div class="modification baseline w-50 panel-column" (click)="setBaselineSelected()"
+          [ngClass]="{'disabled': baselineSelected == false}">
+          <div class="header mx-2">
+            <h3>Baseline</h3>
+          </div>
+          <div class="d-flex flex-column mx-4 mb-4">
+            <div class="p-2">
+              <a class="click-link pull-right small" (click)="addBaselineEquipment()">+Add Equipment</a>
+            </div>
+
+            <div *ngFor="let data of baselineData; let i = index;">
+              <!-- app form -->
+              <app-compressed-air-reduction-form [index]="i" [settings]="settings" [data]="data" [isBaseline]="true"
+                (emitCalculate)="updateBaselineData($event, i)" (emitChangeField)="changeField($event)"
+                (emitRemoveEquipment)="removeBaselineEquipment($event)" [selected]="baselineSelected">
+              </app-compressed-air-reduction-form>
+            </div>
+
+            <div class="w-100 justify content-center btn-group btn-group-sm">
+              <button type="button" class="btn btn-danger pull-right btn-sm btn-margin-10 active"
+                (click)="btnResetData()">
+                Reset Data
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <!-- modification -->
+        <div *ngIf="modificationExists" class="w-50 panel-column modification" (click)="setModificationSelected()"
+          [ngClass]="{'disabled': baselineSelected == true}">
+          <!-- modification header -->
+          <div class="header mx-2">
+            <h3>Modification</h3>
+          </div>
+          <!-- forms -->
+          <div class="d-flex flex-column mx-4 mb-4">
+            <div class="p-2">
```

Comment:
Can we move the add fixture link up into the header? That would maintain consistency with the other calculators. Theres some small css added in the others too

Reply:
Same for baseline